docs/Fachschaftsordnung.tex

```diff
@@ -179,7 +180,7 @@ \section{E-Mail-Verteiler und Nutzergruppe}
 \begin{enumerate}
 	\item Der FSR unterhält einen E-Mail-Verteiler: \textbf{\email{fsphys-l@listserv.uni-muenster.de}}. Dieser wird von Herrn Dr.\ Adam (\email{adamh@uni-muenster.de}) technisch verwaltet. E-Mails an die E-Mail-Adresse der Fachschaft (\email{fsphys@uni-muenster.de}) werden vom Verwalter nach einer Spam-Filterung an den internen Verteiler weitergeleitet. Die interne Kommunikation des FSR erfolgt über den Verteiler.
 	\item Mitglied in dem E-Mail-Verteiler wird jedes Mitglied des FSR. Dem Verwalter wird jeweils mitgeteilt, wie er über einen Aufnahmeantrag zu entscheiden hat.
-	\item Mit Austritt aus dem FSR endet die Mitgliedschaft im E-Mail-Verteiler. Alte Mitglieder können (mündlich) beantragen, weiterhin Mitglied im E-Mail-Verteiler zu bleiben. Der FSR entscheidet über den Antrag.
+	\item Mit Austritt aus dem FSR endet die Mitgliedschaft im E-Mail-Verteiler. Alte Mitglieder können (mündlich) beantragen, für ein weiteres Jahr Mitglied im E-Mail-Verteiler zu bleiben. Der FSR entscheidet über den Antrag.
```

Comment:
Das würde insgesamt nur 1 Jahr bedeuten. Meintest du etwas wie "können für jeweils ein Jahr beantragen"?

Sollten wir den Terminus "Alte Mitglieder" in "ehemalige Mitglieder" überall umändern?